libs/logger/src/logger.service.ts

```
@@ -25,60 +22,96 @@ export default class LoggerService implements Logger {
     this.app = configService.get<string>('APP');
   }
 
-  public log(
-    level: LogLevel,
-    message: string | Error,
-    data?: LogData,
-    profile?: string,
-  ): void {
+  public log(level: LogLevel, message: string | Error, data?: LogData, profile?: string): void {
+    this.emitToOtel(level, message, data);
     return this.logger.log(level, message, this.getLogData(data), profile);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)